airflow/models/taskinstance.py

```diff
@@ -306,6 +306,7 @@ def _run_raw_task(
             ti.state = TaskInstanceState.SKIPPED
             _run_finished_callback(callbacks=ti.task.on_skipped_callback, context=context)
             TaskInstance.save_to_db(ti=ti, session=session)
+            return None
```

Comment:
If you put here the `return` it won't reach these [lines](https://github.com/apache/airflow/blob/main/airflow%2Fmodels%2Ftaskinstance.py#L351-L355), is that okay?

Reply:
It will.
finally block is always executed before the return
https://stackoverflow.com/a/71722877/14624409

Reply:
It's not ok 😄 
Because of this, the tests are failing.

I changed it to run those lines.